easybuild/easyconfigs/c/CatBoost/CatBoost-1.2-gfbf-2023a.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'PythonBundle'
+
+name = 'CatBoost'
+version = '1.2'
+
+homepage = 'https://catboost.ai'
+description = """CatBoost is a high-performance open source library for gradient boosting on decision trees"""
+
+toolchain = {'name': 'gfbf', 'version': '2023a'}
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Clang', '16.0.6'),
+    ('SciPy-bundle', '2023.07'),
+    ('matplotlib', '3.7.2'),
+    ('plotly.py', '5.16.0'),
+    ('graphviz-python', '0.20.1'),
+    ('Ninja', '1.11.1'),
+    ('conan', '1.60.2'),
```

Comment:
Hmm, is `conan` really a *runtime* dependency of `CatBoost`?

That would be strange, since `conan` is a package manager...